### ISSUE TITLE
Composer create project error

### ISSUE THREAD

**Issue by ugintl** [AcroMedia/drupal-orange-project#19]
After running composer create project command, the download was complete but I see this error.
![orange-project-composer-error](https://user-images.githubusercontent.com/12570802/108601884-d20e4e00-73c0-11eb-9c69-5ca012e7d913.png)

**Comment by 7o7o** [comment]
Yeah me too, for some months in my case. I thought i was the only one seeing them haha.
